Makefile

```diff
@@ -69,6 +69,7 @@ test-js:
 test-all: ## run tests on every supported Python/Django combination
 	tox -e quality
 	tox
+	cd edx_proctoring && i18n_tool validate
```

Comment:
Why do we do this in the `Makefile` instead of `.travis.yml` like we're doing in the other repos?

Reply:
@amangano-edx  i have no idea. should i update this.

Reply:
I would make this a separate make target (`make extract_translations`) and then run it in the script component of travis.yml, since that's how it's run in the other repos.